src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -815,7 +815,7 @@ private IDynamicMemberReferenceOperation CreateBoundDynamicMemberAccessOperation
 
         private IDynamicInvocationOperation CreateBoundDynamicCollectionElementInitializerOperation(BoundDynamicCollectionElementInitializer boundCollectionElementInitializer)
         {
-            Lazy<IOperation> operation = new Lazy<IOperation>(() => Create(boundCollectionElementInitializer.ImplicitReceiver));
+            _ = new Lazy<IOperation>(() => Create(boundCollectionElementInitializer.ImplicitReceiver));
```

Comment:
wat...

Reply:
Seems like a bug in operation factory as `operation` is indeed unused: http://source.roslyn.io/#Microsoft.CodeAnalysis.CSharp/Operations/CSharpOperationFactory.cs,fbd99457fe5d4252. Tagging @333fred as he is doing some cleanup in the operation factories.

Reply:
Will be removed when I merge my refactor.